prog/install_rhizome.rb

```diff
@@ -37,8 +37,7 @@ class Prog::InstallRhizome < Prog::Base
 
   label def install_gems
     sshable.cmd("bundle config set --local path vendor/bundle")
-    sshable.cmd("bundle install --gemfile common/Gemfile")
-    sshable.cmd("bundle install --gemfile #{frame["target_folder"]}/Gemfile")
+    sshable.cmd("bundle install")
```

Comment:
Do we want to install gems that required for only vm host machine to postgres vm?

Reply:
No, and we don't install them. Gemfile in parent directory will be combination of common/Gemfile and postgres/Gemfile at that time, because only those folders will be exist in postgres vm

Reply:
Okey. I missed that part. I though we copy all rhizome directory,